rust/src/wrapper/format.rs

```diff
@@ -952,3 +952,27 @@ impl From<FormatScript> for xlsx::FormatScript {
         }
     }
 }
+
+/// The `FontScheme` enum defines the {@link Format} font scheme property.
+///
+#[derive(Clone, Copy, Default)]
+#[wasm_bindgen]
+pub enum FontScheme {
+    /// This is used to indicate a theme font that is used for body text.
+    #[default]
+    Body,
+    /// This is used to indicate a theme font that is used for headings.
+    Headings,
+    /// This is used to indicate a custom font that is not part of the theme.
+    None,
+}
+
+impl From<FontScheme> for xlsx::FontScheme {
+    fn from(scheme: FontScheme) -> xlsx::FontScheme {
+        match scheme {
+            FontScheme::Body => xlsx::FontScheme::Body,
+            FontScheme::Headings => xlsx::FontScheme::Headings,
+            FontScheme::None => xlsx::FontScheme::None,
+        }
+    }
+}
```

Comment:
There is a breaking change in v0.92.0

https://github.com/jmcnamara/rust_xlsxwriter/issues/158#issuecomment-3523691129

Although, I think set_font_scheme is not used as noted

>    /// This function is implemented for completeness but is rarely used in
    /// practice.